install_config/install/advanced_install.adoc

```diff
@@ -201,9 +201,25 @@ All].
 
 |`*openshift_master_named_certificates*`
 .2+.^|These variables are used to configure xref:../../install_config/certificate_customization.adoc#install-config-certificate-customization[custom certificates] which are deployed as part of the installation. See xref:advanced-install-custom-certificates[Configuring Custom Certificates] for more information.
-
 |`*openshift_master_overwrite_named_certificates*`
 
+|`openshift_hosted_registry_cert_expire_days`
+|Validity of the auto-generated registry certificate in days. Defaults to `730` (2 years).
+
+|`openshift_ca_cert_expire_days`
+|Validity of the auto-generated CA certificate in days. Defaults to `1825` (5 years).
+
+|`openshift_node_cert_expire_days`
+|Validity of the auto-generated node certificate in days. Defaults to `730` (2 years).
+
+|`openshift_master_cert_expire_days`
+|Validity of the auto-generated master certificate in days. Defaults to `730` (2 years).
+
+|`etcd_ca_default_days`
+|Validity of the auto-generated external etcd certificates in days. Controls
+validity for etcd CA, peer, server and client certificates. Defaults to `1825`
+(5 years).
```

Comment:
@php-coder @abutcher Is https://github.com/openshift/openshift-ansible/pull/3581#issuecomment-285136322 still relevant? Do any additional caveats need to be mentioned here for `etcd_ca_default_days`?